### PR TITLE
Feature 330

### PR DESCRIPTION
**Pull request by jctrebalag**

### CODE REVIEW

File: src/pages/ni/billing/DebitsArchive.vue [hunk 1]
```diff
@@ -67,7 +72,7 @@ export default {
     },
     async getWithdrawals () {
       try {
-        this.withdrawals = await this.$gdrive.getList({ folderId: process.env.GOOGLE_DRIVE_WITHDRAWAL_FOLDER_ID });
+        this.withdrawals = await this.$gdrive.getList({ folderId: this.user.company.withdrawalFolderId });
```
Comment (sophiemoustard) [inline src/pages/ni/billing/DebitsArchive.vue]: rajouter un check pour verifier que ca existe bien ?

Comment (jctrebalag) [reply src/pages/ni/billing/DebitsArchive.vue]: Yep ! Done !

Comment (calemany) [reply src/pages/ni/billing/DebitsArchive.vue]: `!this.user.company || !this.user.company.withdrawalFolderId`

File: src/pages/ni/billing/DebitsArchive.vue [hunk 2]
```diff
@@ -67,10 +73,12 @@ export default {
     },
     async getWithdrawals () {
       try {
-        this.withdrawals = await this.$gdrive.getList({ folderId: process.env.GOOGLE_DRIVE_WITHDRAWAL_FOLDER_ID });
+        if (!this.user.company && !this.user.company.withdrawalFolderId) return NotifyNegative('Dossier de prélèvement manquant');
```
Comment (calemany) [inline src/pages/ni/billing/DebitsArchive.vue]: `!this.user.company || !this.user.company.withdrawalFolderId`

Comment (jctrebalag) [reply src/pages/ni/billing/DebitsArchive.vue]: Bien vu, c'est corrigé !

**Pull request merged**